provision/rosa-cross-dc/keycloak-benchmark-crossdc-tests/src/test/java/org/keycloak/benchmark/crossdc/DatacenterConfig.java

```diff
@@ -0,0 +1,63 @@
+package org.keycloak.benchmark.crossdc;
+
+public class DatacenterConfig {
```

Comment:
Might be better to just have classic objects initialized once via constructor instead of constantly resetting this static stuff throughout the test. Initialize the objects in the setup method and then just use `dc1.getLoginEndpoint()`, `dc2.getISPNEnpoint()`, etc. from the test method.